### ISSUE TITLE
multi-label - 多标签 - Classification - Segmentation

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#693]
# Reference
- [ ] [paper - A Review on Multi-Label Learning Algorithms](https://cs.nju.edu.cn/zhouzh/zhouzh.files/publication/tkde13rev.pdf)
- [ ] [论文笔记：多标签学习综述（A review on multi-label learning algorithms）](https://www.cnblogs.com/liaohuiqiang/p/9339996.html)
- [ ] [multi-label image classification:多标签图像分类总结](https://www.cnblogs.com/youyou0/p/10250326.html)
- [ ] [【技术综述】多标签图像分类综述](https://zhuanlan.zhihu.com/p/70449858)
- [ ] [A Tutorial on Multilabel Learning.pdf](https://github.com/junxnone/tech-io/files/4373791/A.Tutorial.on.Multilabel.Learning.pdf)
- [ ] [Multi-label classification - wikipedia](https://en.wikipedia.org/wiki/Multi-label_classification)
- [ ] [ Multiclass and multilabel algorithms - sklearn](https://scikit-learn.org/stable/modules/multiclass.html)
- [ ] [分类问题-----多标签（multilabel）、多类别（multiclass）](https://blog.csdn.net/qq_27009517/article/details/80264919)
- [ ] [深度学习在多标签分类中的应用](https://blog.csdn.net/littlely_ll/article/details/81053928)

# Brief
## Segmentation

# Metrics
- AP
- mAP
- 汉明距离 - Hamming loss
- 错误率
- 覆盖率
- 排序损失

---
- 基于样例（Example-based）
- 基于标签（Label-based）

# Datasets
- [ ] [Tencent ML-Images](https://github.com/Tencent/tencent-ml-images/)
- [ ] Pascal VOC
- [ ] COCO

**Comment by junxnone** [comment]
#491
